{"name": "address中city不输入新增失败", "status": "passed", "attachments": [{"name": "stdout", "source": "baa9c880-0106-43ff-b8f6-dd6a04281702-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '76', 'casename': 'address中city不输入新增失败', 'mode': '地址管理', 'data': '{\"fistname\":\"fistname\",\"lastname\":\"lastname\",\"contactnumber\":\"contactnumber\",\"coordinated_area1\":\"coordinated_area1\",\"coordinated_area2\":\"\",\"coordinated_area3\":\"C2\",\"coordinated_postalCode\":\"coordinated_postalCode\",\"coordinated_addressLine\":\"coordinated_addressLine\"}', 'assert_way': 'class name=ant-form-explain', 'result': 'Please input city!'}"}], "start": 1651196104138, "stop": 1651196118770, "uuid": "b5249924-9733-4a09-9267-44f84b3f6ebc", "historyId": "cb8b8c1d31541ac25c02965907940e7d", "testCaseId": "840f25e4c9ed223ada744fce7d35f48a", "fullName": "test_main.Test_Main#test_address", "labels": [{"name": "feature", "value": "地址管理"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "23236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}